lib/mutant/mutator/node/begin.rb

```diff
@@ -13,23 +13,10 @@ class Begin < self
         #
         # @return [undefined]
         def dispatch
-          Util::Array.each(children, self, &method(:emit_child_subset))
-          children.each_with_index do |child, index|
-            mutate_child(index)
-            emit(child) unless children.eql?([child])
+          mutate_single_child do |child|
```

Comment:
@dkubb Proactive comment: `mutate_single_child` yields 2 args, so `&method(:emit)` is not possible here.